extensions/amp-bind/0.1/amp-state.js

```diff
@@ -135,22 +150,39 @@ export class AmpState extends AMP.BaseElement {
       (getSourceOrigin(src) == getSourceOrigin(ampdoc.win.location))) {
       policy = UrlReplacementPolicy.ALL;
     }
-    return batchFetchJsonFor(
-        ampdoc, element, /* opt_expr */ undefined, policy, opt_refresh);
+
+    // If cross-origin="amp-viewer-auth-token-via-post" attribute is present,
+    // the viewer will make a remote xhr POST request with an auth token in the
+    // request body. Requires amp-viewer-assistance extension for auth token.
+    const crossOriginAttr = element.getAttribute('cross-origin');
+    if (crossOriginAttr &&
+        crossOriginAttr.indexOf('amp-viewer-auth-token-via-post') >= 0) {
```

Comment:
Why `indexOf` vs. `trim()` and `===`? 

If multiple keywords are meant to be allowed, we could do `attr.split(' ').map(s => s.trim()).includes(...)`.

Reply:
Done.